app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt

```diff
@@ -4,11 +4,11 @@ import android.content.Context
 import android.view.View
 import android.view.inputmethod.InputMethodManager
 
-/** KeyboardHelper helps to change the visibility of softinputkeybord. */
+/** KeyboardHelper helps to change the visibility of SoftInputKeyboard. */
```

Comment:
``SoftInputKeyboard`` isn't a class. Suggest writing it out, instead: ``the soft input keyboard``. Ditto below.